ext/standard/tests/array/range/range_inputs_string_digits.phpt

```diff
@@ -0,0 +1,97 @@
+--TEST--
+Test range() function digits
```

Comment:
Such a test exists in ``range_inputs_numeric_basic.phpt``

Reply:
I've split them up into more files

Reply:
Sorry for the confusion. The description is correct, however I meant test cases like range("12","A"). In your implementation it returns:

```
Warning: range(): Argument #1 ($start) must be a string if argument #2 ($end) is a string, argument #2 ($end) converted to 0 in Command line code on line 1
array(13) {
  [0]=>
  int(12)
  [1]=>
  int(11)
  [2]=>
  int(10)
  [3]=>
  int(9)
  [4]=>
  int(8)
  [5]=>
  int(7)
  [6]=>
  int(6)
  [7]=>
  int(5)
  [8]=>
  int(4)
  [9]=>
  int(3)
  [10]=>
  int(2)
  [11]=>
  int(1)
  [12]=>
  int(0)
}

```

The logic behind is arguable. I'm not sure exactly how it should behave for such case, but for sure the description is kind misleading. Because both of the arguments are string already.

Reply:
``range()`` can only work with single byte strings. Thus the behaviour of trying to interpret the numeric string as an int is the correct one. I will amend the error message to precise "single byte strings"